plugins/stashNotes/stashNotes.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+(function () {
+    const api = window.PluginApi;
+    const React = api.React;
+    const { Button, Modal } = api.libraries.Bootstrap;
+    const { faNoteSticky } = api.libraries.FontAwesomeSolid;
+    const NotesComponent = () => {
+        const storageKey = 'Stash Notes';
+        const [display, setDisplay] = React.useState(false);
+        const [notes, setNotes] = React.useState('');
+        const enableModal = () => setDisplay(true);
+        const disableModal = () => setDisplay(false);
+        const saveNotes = (notes) => {
+            localStorage.setItem(storageKey, notes);
+            disableModal();
+        };
+        React.useEffect(() => {
+            const notesFromStorage = localStorage.getItem(storageKey);
+            if (notesFromStorage) {
+                setNotes(notesFromStorage);
+            }
+        }, []);
+        return (React.createElement(React.Fragment, null,
+            React.createElement(NavButton, { onClickHandler: enableModal }),
+            React.createElement(NotesModal, { displayState: display, onCloseHandler: disableModal, onSaveHandler: saveNotes, notesState: notes, notesChangeHandler: (n) => setNotes(n) })));
+    };
+    const NavButton = ({ onClickHandler }) => {
+        const { Icon } = api.components;
+        return (React.createElement(React.Fragment, null,
+            React.createElement(Button, { className: "minimal d-flex align-items-center h-100", title: "Notes", onClick: onClickHandler },
```

Comment:
Right now the button is misplaced compared to other buttons. Changing the class to `nav-utility minimal btn btn-primary` fixes the alignment.